bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -511,7 +571,7 @@ impl Vm {
     /// # Error
     ///
     /// If fail to load, then an error is returned.
-    pub fn load_wasm_from_module(&mut self, module: &Module) -> WasmEdgeResult<()> {
+    pub fn load_wasm_from_module(&self, module: &Module) -> WasmEdgeResult<()> {
         unsafe {
```

Comment:
Please still use '&mut selfas the design of the currentsys` APIs tried to "express" the mutability of the relevant C-API.